src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js

```diff
@@ -72,6 +77,46 @@ const InstanceContributorsView = ({
     );
   }, [contributors, contributorTypes, contributorNameTypes]);
 
+  const getName = (item) => {
+    if (segment === segments.instances && source === 'MARC' && 'item.authorityId') {
```

Comment:
I think `item.authorityId` shouldn't be a string here, right?

Reply:
Thanks, fixed